.github/workflows/snap.yaml

```diff
@@ -8,18 +8,23 @@ on:
       - noetic-devel
   workflow_dispatch:
   workflow_call:
+   inputs:
+      branch-name:
+        required: false
+        type: string
+        default: ''
 
 jobs:
   snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
+    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@fix/default_branch_name
     secrets:
       store-login: ${{ secrets.STORE_LOGIN }}
     with:
       ubuntu-image: ubuntu-20.04
-      branch-name: noetic-devel
+      branch-name: ${{ inputs.branch-name == '' && github.ref || inputs.branch-name }}
       snap-name: turtlebot3c
       snap-install-args: --dangerous
-      publish: true
+      publish: ${{ github.event_name == 'push' || github.event_name == 'workflow_call'}}
```

Comment:
why do we push also on the event that the workflow has been called, would that publish also for instance on a monthly check?

Reply:
Yes. And that is the idea. So the snaps available on the store are up to date.